# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/webrtc/transport/test/webrtcproxychannel_unittest.cpp
# Commit: 521c9e019678
# Full Hash: 521c9e019678f584590f63473416d40e03f33342
# Author: Valentin Gosu <valentin.gosu@gmail.com>
# Date: 2023-03-15 04:10:55
# Regressor Bug: 1805372
# File Overlap Count: 1
# Description:
#   Bug 1805372 - Pass effectiveTRRMode to nsHttpChannel from socketTransport/DnsAndConnectSocket r=necko-reviewers,kershaw
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D164539
# ==============================================================================

diff -r dfb39ca05529 -r 521c9e019678 dom/media/webrtc/transport/test/webrtcproxychannel_unittest.cpp
--- a/dom/media/webrtc/transport/test/webrtcproxychannel_unittest.cpp	Tue Mar 14 15:29:49 2023 +0200
+++ b/dom/media/webrtc/transport/test/webrtcproxychannel_unittest.cpp	Tue Mar 14 12:59:40 2023 +0000
@@ -185,6 +185,15 @@
     MOZ_ASSERT(false);
     return NS_OK;
   }
+  NS_IMETHOD GetEffectiveTRRMode(
+      nsIRequest::TRRMode* aEffectiveTRRMode) override {
+    MOZ_ASSERT(false);
+    return NS_OK;
+  }
+  NS_IMETHOD GetTrrSkipReason(nsITRRSkipReason::value* aSkipReason) override {
+    MOZ_ASSERT(false);
+    return NS_OK;
+  }
   NS_IMETHOD GetRetryDnsIfPossible(bool* aRetryDns) override {
     MOZ_ASSERT(false);
     return NS_OK;